providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdProviderTest.java

```diff
@@ -863,4 +912,29 @@ private FlagdProvider createProvider(GrpcConnector grpc, Cache cache, Supplier<P
         return provider;
     }
 
+    // Create an in process provider
+    private FlagdProvider createInProcessProvider() {
+
+        final FlagdOptions flagdOptions = FlagdOptions.builder()
+                .resolverType(Config.Resolver.IN_PROCESS)
+                .deadline(1000)
+                .build();
+        final FlagdProvider provider = new FlagdProvider(flagdOptions);
+        final MockStorage mockStorage = new MockStorage(new HashMap<String, FeatureFlag>(), new LinkedBlockingQueue<StorageState>(List.of(StorageState.OK)));
```

Comment:
`List.of` won't work as we still maintain support for Java 8

Reply:
Fixed.